File: College-Sports-WebApp/ClientApp/src/api/models/policy.ts
/* tslint:disable */
/* eslint-disable */
import { ViewingPolicy } from '../models/viewing-policy';
export interface Policy {
  createdBy?: string | null;
  createdOn?: string;
  id?: string | null;
  lastModifiedBy?: string | null;
  lastModifiedOn?: string;
  viewingPolicies?: Array<ViewingPolicy> | null;
}

File: College-Sports-Domain/DTO/Policy.cs
using System.Collections.Generic; 
using System;
using System.Text.Json.Serialization;
namespace College_Sports_WebApp.Database.Models{ 

    public class Policy
    {
        public List<ViewingPolicy> viewingPolicies { get; set; }
        public string id { get; set; }
        public string createdBy { get; set; }
        public DateTime createdOn { get; set; }
        public string lastModifiedBy { get; set; }
        public DateTime lastModifiedOn { get; set; }
    }

}